### PR TITLE
Including 3 more operations

### PR DESCRIPTION
**Pull request by abramvandergeest**
ifin
ifnotin
getStopWords

This is intended to make nlp demo/work easier

### CODE REVIEW

File: operations/cleaning/ifnotin/operation.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+type Operation struct {
+	logger log.Logger
+}
+
+func New(ctx operation.InitContext) (operation.Operation, error) {
+	return &Operation{logger: ctx.Logger()}, nil
+}
+
+func (a *Operation) Eval(inputs map[string]interface{}) (interface{}, error) {
+	fmt.Println("JKSDHFSFLKDJSf")
```
Comment (skothari-tibco) [inline operations/cleaning/ifnotin/operation.go]: Is it supposed to be here ?

File: operations/cleaning/ifnotin/operation.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+	input.FromMap(inputs)
+
+	a.logger.Debug("inputs", inputs)
+	a.logger.Info("Executing operation ifnotin ...")
+
+	arr0 := inputs["arr0"].([]string)
+	arr1 := inputs["arr1"].([]string)
+
+	fmt.Println("London")
```
Comment (skothari-tibco) [inline operations/cleaning/ifnotin/operation.go]: This one too?

Comment (abramvandergeest) [reply operations/cleaning/ifnotin/operation.go]: Removed it and pushed change

File: operations/cleaning/ifnotin/operation.go (hunk 1 above)
Comment (abramvandergeest) [reply operations/cleaning/ifnotin/operation.go]: Removed it and pushed change

**Review by skothari-tibco — APPROVED** [review]

**Review by SteveNY-Tibco — COMMENTED** [review]
looks good.

**Comment by SteveNY-Tibco** [comment]
l

**Comment by SteveNY-Tibco** [comment]
looks good.

**Pull request merged**